README.md

```diff
@@ -55,8 +55,8 @@ then add this interpreter by going to the `File->Settings` in PyCharm, then `Pro
 click the cog on the right side to add an existing interpreter and select `Conda` and `Python 3.8 (mantidnightly)`.
 Then go to `Run -> Edit Configurations` and create new configuration with this interpreter.
 Specify `start_mslice.py` as the startup script.
```

Comment:
Might be worth adding that `src` now needs to be added as a `sources root` or have the working directory at `<mslice root>\src` rather than `<mslice root>` as per the image.

Reply:
Good point, I will just replace the image.